charts/node-specific-sizing/templates/mutatingadmissionwebhook.yaml

```diff
@@ -0,0 +1,27 @@
+kind: MutatingWebhookConfiguration
+apiVersion: admissionregistration.k8s.io/v1
+metadata:
+  name: {{ include "node-specific-sizing.fullname" . }}
+  annotations:
+    cert-manager.io/inject-ca-from: kube-system/{{ include "node-specific-sizing.fullname" . }}-client-cert
+webhooks:
+  - name: {{ include "node-specific-sizing.fullname" . }}.svc.cluster.local
+    objectSelector:
+      matchLabels:
+        node-specific-sizing.manomano.tech/enabled: "true"
+    admissionReviewVersions: [ "v1" ]
+    sideEffects: None
+    failurePolicy: Ignore
+    timeoutSeconds: 2
```

Comment:
I recommend a 1 second timeout here, and it could be worth making this customizable through values.